Update: Ad Portal Flow Landing and Confirmation Page

jennsun opened the pull request:
https://app.clickup.com/t/1b33hvp

Created new landing and confirmation pages for the "Ad Portal" flow of the SDK Demo Site. 
Currently, the Ad Portal flow is not updated with the new pages -- the base of the pages is just created. 
- The "Select your template" button on the Landing Page does not have any functionality but will eventually redirect to the Ad Portal Template browser. 
- The "Restart Demo" button on the Confirmation page redirects to the Configuration page of the Demo site with the cards/options of navigation flows.

jennsun committed:
Update: Ad Portal Flow Landing and Confirmation Page

simanthastew commented on example_site/src/components/AdPortalConfirmation.js:
```diff
@@ -0,0 +1,37 @@
+        setShowLandingPage,
+    } = useAppContext();
+
+    async function onResetWaymarkInstance() {
```
Looks like we're using this in the `AccountPage` component as well - I think we can add this to `AppProvider` and import it via `useAppContext`

simanthastew commented on example_site/src/components/AdPortalConfirmation.js:
```diff
@@ -0,0 +1,37 @@
+            />
+            <button 
+                className="submit-button"
+                onClick={onResetWaymarkInstance}>
```
The ending closure should always be at the same indentation level as the starting closure, i.e.
```
<button
    className=""
    onClick={() => {}}
>
```

simanthastew reviewed (changes requested):
Nice work! Two very small comments, feel free to merge when you're done

jennsun committed:
PR Review:
Reset custom form on configuration page, made onResetWaymarkInstance an AppProvider function to pass down, cleaned up styling
